packages/react-instantsearch-hooks-dom/src/ui/SearchBox.tsx

```diff
@@ -9,18 +9,32 @@ import type {
   JSXElementConstructor,
 } from 'react';
 
+export type SearchBoxTranslations = {
+  /**
+   * The alternative text of the submit button.
+   */
+  submitTitle: string;
+  /**
+   * The alternative text of the reset button.
+   */
+  resetTitle: string;
+  /**
+   * The placeholder of the search input.
+   */
+  placeholder: string;
+};
+
 export type SearchBoxProps = React.HTMLAttributes<HTMLDivElement> & {
   inputRef: RefObject<HTMLInputElement>;
   isSearchStalled: boolean;
   onChange(event: ChangeEvent): void;
   onReset(event: FormEvent): void;
   onSubmit?(event: FormEvent): void;
-  placeholder?: string;
```

Comment:
this change makes it so the placeholder is no longer customizable by users, as translations aren't exposed. IMO a regular dom prop shouldn't be a translation

Reply:
Fair point. I changed it back in 6d2940b0.